core/utils_test.py

```diff
@@ -794,3 +794,85 @@ def test_url_open(self) -> None:
         self.assertEqual(response.getcode(), 200) # type: ignore[attr-defined]
         self.assertEqual(
             response.url, 'http://www.google.com') # type: ignore[attr-defined]
+
+    def test_get_random_int(self) -> None:
+        self.assertLess(utils.get_random_int(5), 5)
+        self.assertGreaterEqual(utils.get_random_int(5), 0)
+        self.assertLess(utils.get_random_int(True), 1)
+        self.assertGreaterEqual(utils.get_random_int(True), 0)
+        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
+            AssertionError, 'Only positive integers allowed'):
+            utils.get_random_int(-1)
+
+    def test_get_random_choice(self) -> None:
+        list_instance = [1, 5, 9, 11, 15]
+        list_instance2: List[str] = []
+        self.assertIn(utils.get_random_choice(list_instance), list_instance)
+        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
+            AssertionError, 'Only non-empty lists allowed'):
+            utils.get_random_choice(list_instance2)
+
+    def test_get_human_readable_time_string(self) -> None:
+        """Should we test for negative time."""
+        self.assertEqual('December 12 06:42:12', utils.get_human_readable_time_string(944980932342.38))  # pylint: disable=line-too-long
```

Comment:
I think you can break the line instead of having the pylint ignore comment:

```suggestion
        self.assertEqual(
            'December 12 06:42:12',
            utils.get_human_readable_time_string(944980932342.38)
        )
```

ditto for the other pylint line length ignores below

Reply:
Done